server/controllers/reports_proposed/data/debitor_group_report.js

```diff
@@ -0,0 +1,77 @@
+// reports_proposed/data/balance_sheet.js
+// Collects and aggregates data for the enterprise balance sheet
+var q       = require('q');
+var db      = require('../../../lib/db');
+var numeral = require('numeral');
+var sanitize = require('../../../lib/sanitize');
+
+var formatDollar = '$0,0.00';
+var debitorGroupReportDate = new Date();
+
+exports.compile = function (options) {
+  'use strict';
+
+  var i18nDebitorGroupReport = options.language == 'fr' ? require('../lang/fr.json').DEBITOR_GROUP_REPORT : require('../lang/en.json').DEBITOR_GROUP_REPORT;
+  var deferred = q.defer(), context = {}, params = options.involveJournal === true ? [options.dg.account_id, options.dg.account_id] : [options.dg.account_id];
+
+  var sql2 =
+    'UNION ALL SELECT `s`.`uuid` as `saleUUID`, CONCAT(`pr`.`abbr`, `s`.`reference`) AS `saleNumber`, `s`.`cost`, DATE_FORMAT(`s`.`invoice_date`, \'%d-%m-%Y\') AS `invoice_date`, `dg`.`name`, `ac`.`account_txt`, SUM(`pj`.`debit_equiv`) AS `debit`, SUM(`pj`.`credit_equiv`) AS `credit`, `pj`.`trans_id`, ' +
```

Comment:
We can get around that by always name spacing (which is clearer anyway).  For example: `SELECT table.column` rather than `SELECT column`.